website/src/content/docs/analyzer/index.mdx

```diff
@@ -23,12 +23,15 @@ This feature is enabled by default but can be opted-in/out via configuration:
 
 Import statements are sorted by "distance". Modules that are "farther" from the user are put on the top, modules "closer" to the user are put on the bottom:
 
-1. built-in Node.js modules that are explicitly imported using the `node:` protocol;
-2. modules imported via `npm:` protocol. This is a valid syntax when writing code run by Deno, for example;
-3. modules imported via URL;
-4. modules imported from libraries;
-5. modules imported via relative imports;
-6. modules that couldn't be identified by the previous criteria;
+1. modules imported via `bun:` protocol. This is a valid syntax when writing code run by Bun;
+2. built-in Node.js modules that are explicitly imported using the `node:` protocol;
+3. modules imported via `npm:` protocol. This is a valid syntax when writing code run by Deno, for example;
+4. modules imported via URL;
+5. modules imported from libraries;
+6. modules imported via absolute imports;
+7. modules imported from a name prefixed by `#`. This is a valid syntax when [writing code run by Node](https://nodejs.org/api/packages.html#subpath-imports);
+8. modules imported via relative imports;
+9. modules that couldn't be identified by the previous criteria;
```

Comment:
```suggestion
1. modules imported via `bun:` protocol. This is applicable when writing code run by Bun;
2. built-in Node.js modules that are explicitly imported using the `node:` protocol;
3. modules imported via `npm:` protocol. This is applicable when writing code run by Deno, for example;
4. modules imported via URL;
5. modules imported from libraries;
6. modules imported via absolute imports;
7. modules imported from a name prefixed by `#`. This is applicable when [writing code run by Node](https://nodejs.org/api/packages.html#subpath-imports);
8. modules imported via relative imports;
9. modules that couldn't be identified by the previous criteria;
```

I think "valid syntax" is not entirely correct (I know I am the one who wrote it 😅 )